frontend/src/static/js/components/webstatus-overview-table.ts

```diff
@@ -91,6 +91,9 @@ export class WebstatusOverviewTable extends LitElement {
         .browser-impl-unavailable img {
           filter: grayscale(1);
           opacity: 50%;
+          /*  Needed so that the image will stay behind the dialog box */
```

Comment:
I am surprised that anything would interfere with a dialog.

Reply:
Me too. It was interesting to see it. Only the grayscale images came through the dialog. The full color availability images did not